source/module_parameter/input_parameter.h

```diff
@@ -586,5 +586,12 @@ struct Input_para
     int  test_pp = 0;                ///< variables for test_pp only
     int  test_relax_method = false;  ///< variables for test_relax_method only
     int  test_deconstructor = false; ///< variables for test_deconstructor only
+
+    // ==============   #Parameters (21.RDMFT) =====================
+    // RDMFT    jghan added on 2024-07-06
+    std::string ab_initio_type = "ksdft";       // "rdmft", reduced density matrix funcional theory
```

Comment:
not sure we should add this parameter

Reply:
Personally speaking, it's not a good idea to add an option under `esolver_type`, because this PR does not add a new esolver type... In contrast, adding a new parameter helps to reduce code-coupling.

Reply:
If rdmft inherits esolver_ks_lcao, and then we just have a new esolver type?

Reply:
Yes. After that, we can delete this new parameter.